src/client/app/services/monaco.service.ts

```diff
@@ -132,7 +126,7 @@ export class MonacoService {
     private async _registerLanguageServices() {
         let monaco = await MonacoService.current;
         monaco.languages.register({ id: 'libraries' });
-        monaco.languages.setMonarchTokensProvider('libraries', {
+        monaco.languages.setMonarchTokensProvider('libraries', <monaco.languages.IMonarchLanguage>{
```

Comment:
we have monaco.languages.IMonarchLanguage defined in monaco.d.ts. Is there a typescript definition node package we could use instead? Do they have new definitions to update to if we manually replace this file.